Tests/UnitTests/Core/Fitter/KalmanFitterTests.cpp

```diff
@@ -113,10 +113,14 @@ struct MeasurementCreator {
         auto lResolution = vResolution->second.find(layerID);
         if (lResolution != vResolution->second.end()) {
           // Apply global to local
-          Acts::Vector2D lPos;
-          surface->globalToLocal(state.geoContext,
-                                 stepper.position(state.stepping),
-                                 stepper.direction(state.stepping), lPos);
+          Acts::Vector2D lPos{0., 0.};
+          auto lpResult = surface->globalToLocal(
+              state.geoContext, stepper.position(state.stepping),
+              stepper.direction(state.stepping));
```

Comment:
Since this is a test, you can assign directly to `lPos` via `.globalToLocal(...).value()`. This throws an exception if there is an error instead of silently hiding it (as would be the current behavior).